{
    "type": "bundle",
    "id": "bundle--71166b5e-9ae3-4cce-9f43-2cba80e9ba0f",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--1a423122-8c7b-4b0d-a338-fc08bb9118ec",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-04-03T00:20:54.193231Z",
            "modified": "2025-04-03T00:20:54.193231Z",
            "name": "CVE-2025-2998",
            "description": "A vulnerability was found in PyTorch 2.6.0. It has been declared as critical. Affected by this vulnerability is the function torch.nn.utils.rnn.pad_packed_sequence. The manipulation leads to memory corruption. Local access is required to approach this attack. The exploit has been disclosed to the public and may be used.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-2998"
                }
            ]
        }
    ]
}